packages/core/src/browser/storage-service.ts

```diff
@@ -45,10 +46,13 @@ export class LocalStorageService implements StorageService {
     private storage: LocalStorage;
 
     constructor(
-        @inject(ILogger) protected logger: ILogger
+        @inject(ILogger) protected logger: ILogger,
```

Comment:
Use `property injection` rather than `constructor injection` for new dependencies.